pom.xml

```diff
@@ -17,7 +18,7 @@
 
     <artifactId>Office-365-Connector</artifactId>
     <name>Office 365 Connector</name>
-    <version>4.5-SNAPSHOT</version>
+    <version>4.6</version>
```

Comment:
why?

Reply:
clearly rolling in own releases 😞

Reply:
We needed a different version for our jenkins instances. Was not sure how to do it the right way so just changed it here. Also didn't know that commits made after pull request are "added" after this pull request is made.

Reply:
mvn package will generate a hpi, that you can take to your Jenkins instance 👍 and you shouldn't have to change the version